src/ops/style.rs

```diff
@@ -23,7 +23,7 @@ use conrod::widget::Button;
 /// butan.set(buttid, &mut ui_widgets);
 /// # }
 /// ```
-pub fn set_button_style<'a, S>(button: &mut Button<'a, S>) {
+pub fn set_button_style<S>(button: &mut Button<S>) {
```

Comment:
This is lifetime elision. There are two rules:

1. If there all argument lifetimes are different and there's no return lifetime, the lifetimes can be elided.
2. If there is exactly one arg lifetime and one return lifetime, the lifetime can be elided.

Note that defaults for e.g. `Box` (which includes a `'static` lifetime bound unless otherwise stated) may muddle things somewhat.